src/context.ts

```diff
@@ -66,6 +66,7 @@ export function createContext<T extends Record<string, unknown>>(
       service: '',
       path: '',
       operation: '',
+      callback: '',
```

Comment:
```suggestion
```
not used

Reply:
I removed this, but since I had updated the type, I had to change to use https://github.com/stoplightio/http-spec/pull/211/files#diff-c54113cf61ec99691748a3890bfbeb00e10efb3f0a76f03a0fd9ec49072e410aR42 and https://github.com/stoplightio/http-spec/pull/211/files#diff-c54113cf61ec99691748a3890bfbeb00e10efb3f0a76f03a0fd9ec49072e410aR41 to ensure the types resolved properly.

Reply:
this was still removed as part of switching to the omit pattern